README.rst

```diff
@@ -563,6 +563,21 @@ their hardware ID instead of their name.
 
 If you still want to reference devices by their name, set this to ``False``.
 
+``OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME``
```

Comment:
Please update your commit message. your are talking of `OPENWISP_DEVICE_VERBOSE_NAME` in your commit message. it should be `OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME` as here